packages/webapp/components/search/SearchV1Page.tsx

```diff
@@ -84,13 +87,22 @@ const SearchPage = (): ReactElement => {
     >
       <NextSeo {...seo} />
       {(!!content || !!data) && (
-        <SearchResult
-          queryKey={queryKey}
-          isInProgress={isLoading}
-          chunk={chunk}
-          searchMessageProps={{ isLoading }}
-          className="mt-6"
-        />
+        <div>
+          <SearchResult
+            queryKey={queryKey}
+            isInProgress={isLoading}
+            chunk={chunk}
+            searchMessageProps={{ isLoading }}
+            className="mt-6"
+          />
+          <SearchProviderButton
+            className="order-4 mt-4"
+            provider={SearchProviderEnum.Posts}
+            query={query}
+          >
+            <span>Search posts on daily.dev instead</span>
```

Comment:
Same here I think.

Reply:
Nope, its text like this when on AI search page 
<img width="454" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/942161bc-9310-4343-8ab7-b8370941643e">

Reply:
Oh sorry misread where this was implemented then